CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| | Update fake GMI to remove cookies checking | |
```

Comment:
I guess we will also need to do a release of old Genie and pull the updated version into `childrens-games-genie-starter-pack`.